Tests/StreamChatTests/Controllers/ChannelController/ChannelController_Tests.swift

```diff
@@ -1734,7 +1734,7 @@ final class ChannelController_Tests: XCTestCase {
         // Unlike new DM ChannelController, this ChannelController knows it's final `cid` so it should be able to fetch initial values
         // from DB, without the `synchronize` call
         // Assert that initial reported values are correct
-        XCTAssertEqual(controller.channel?.cid, dummyChannel.channel.cid)
+        AssertAsync.willBeEqual(controller.channel?.cid, dummyChannel.channel.cid)
```

Comment:
All of these others are related to the fact that background mapping was turned on and now controllers delay their updates. Not sure why timing issues appeared now, maybe the DB change changed timings.